### ISSUE TITLE
Support DOI overhaul - multivalued award DOIs

### ISSUE THREAD

**Issue by naglepuff** [microbiomedata/nmdc-server#1010]
`nmdc-schema` is retiring the DOI slot in favor of more specific, multivalued slots e.g. `award_dois`.

The data portal should be updated to reflect this.

A rough outline of the required changes are:

- [ ] Retire column `doi` of table `study`
- [ ] Model the new relationships with SQLAlchemy
- [ ] Update ingest to populate the new `study_award_doi_association` table
- [ ] Update the web server schemas (`Pydantic` and `typescript`) 
- [ ] Update the study landing page

**Comment by naglepuff** [comment]
Note that in order to really complete this task, microbiomedata/nmdc-schema#1039 must be merged and mongo data must be migrated.

Kitware can start development and fake test data initially.

Mongo data migration should be done first in dev. Prod should remain unchanged until we are sure that we've completed the task. This is to minimize risk under our current deadlines.

**Comment by mslarae13** [comment]
Checked in with @marySalvi  some of this has been done. There's a new task of getting ESS-Dive and Massive DOIs in

**Comment by ssarrafan** [comment]
@naglepuff @marySalvi @jeffbaumes is this actively being worked on in the next 2 weeks?

**Comment by ssarrafan** [comment]
Assuming this is active per the study page refresh board.

**Comment by mslarae13** [comment]
@ssarrafan yes, @marySalvi has been working on this. I think it's getting close!
